src/connectors/toggle-refinement/connectToggleRefinement.js

```diff
@@ -135,6 +138,32 @@ export default function connectToggleRefinement(renderFn, unmountFn = noop) {
       },
 
       init({ state, helper, createURL, instantSearchInstance }) {
+        sendEvent = (...args) => {
+          if (args.length === 1) {
+            instantSearchInstance.sendEventToInsights(args[0]);
+            return;
+          }
+          const [eventType, isRefined, eventName = 'Filter Applied'] = args;
+          if (eventType !== 'click' || on === undefined) {
+            return;
+          }
+          // Checking
+          if (!isRefined) {
+            instantSearchInstance.sendEventToInsights({
+              insightsMethod: 'clickedFilters',
+              widgetType: 'ais.toggleRefinement',
```

Comment:
use this.$$type

Reply:
https://github.com/algolia/instantsearch.js/pull/4463/commits/5cf4020b26e6d80c161ad914e65d5740c98316cf